README.md

```diff
@@ -149,12 +149,11 @@ This module and the object it creates `CoinInfo<T>` will tell the ITS the inform
 
 This module is responsible for creating tokenIds for both registerred and 'unregistered' (coins that are given to the ITS expecting a remote incoming `DEPLOY_INTERCHAIN_TOKEN` message) coins. We might just go back to using addresses because the UX would slightly improve, but doing it this way improves code readablility and is more in line with what Sui tries to do.
 
-### Storage
-
-This module is responsible for managing all of the storage needs of the ITS
-
 ### Service
 
 This is the module that anyone would directly interract with. It needs to be able to do the following
 
-- `register_coin<T>`: This function takes the `ITS` object and mutates it by adding a coin with the specified `CoinManagement<T>` and `CoinInfo<T>`.
+- `register_coin_from_info<T>`: This function takes the `InterchainTokenService` object and mutates it by adding a coin with the specified name, symbol, decimals, and `CoinManagement<T>`.
+- `register_coin_from_metadata<T>`: This function takes the `InterchainTokenService` object and mutates it by adding a coin with the specified `CoinMetadata<T>` and `CoinManagement<T>`.
+- `register_coin_metadata`: This function takes the `CoinMetadata<T>` for a token, and prepares a message for Axelar Hub to register the coin's decimal precision and Sui coin type. The returned `MessageTicket` can be broadcast to the `Gateway` contract to enable linking the coin as an interchain token.
+- `link_coin`: This function takes the token linking parameters (`Channel`, salt, destination token, chain and `TokenManagerType`) for a registered coin type, and prepares a `Gateway` message for Axelar Hub. The returned `MessageTicket` can be broadcast to the `Gateway` contract to link the coin to the destination token via Axelar Hub.
```

Comment:
Missing new line at the end. Update your editor config to not remove those since we follow this pattern in all repos?